packages/itwinui-css/src/input/input.scss

```diff
@@ -57,6 +57,8 @@
     cursor: not-allowed;
     --_hover-color: var(--iui-color-background-disabled);
   }
+
+  @include textarea;
```

Comment:
Can we apply this only for `textarea` elements? Looks invalid, when we have `resize` prop on `input`.
We can use your loved `:where` to keep specificity down.

Reply:
Trash in css, which is not needed

Reply:
This looks like a lot more "trash" to me
![](https://user-images.githubusercontent.com/9084735/185147359-e7dc8cd8-5c49-4414-a574-931971bc2ca5.png)

but changed

Reply:
Something got messed up with lock file?

Reply:
i just ran `yarn` when i merged v1 branch here. maybe it was messed up in the v1 branch

Reply:
pushed updated lockfile in v1 branch so it should not show changed here anymore